src/main/java/org/jabref/gui/help/AboutDialogController.java

```diff
@@ -1,8 +1,11 @@
 package org.jabref.gui.help;
 
+
```

Comment:
That is one empty line too much, which is also one of the reasons why the travis build fails